.vscode/settings.template.json

```diff
@@ -54,26 +55,15 @@
   "[makefile]": {
     "editor.insertSpaces": false
   },
-  "python.testing.pytestEnabled": true,
-  "autoDocstring.docstringFormat": "pep257",
   "hadolint.hadolintPath": "${workspaceFolder}/scripts/hadolint.bash",
   "hadolint.cliOptions": [],
-  "shellcheck.executablePath": "${workspaceFolder}/scripts/shellcheck.bash",
-  "shellcheck.run": "onSave",
-  "shellcheck.enableQuickFix": true,
-  "python.formatting.provider": "black",
-  "isort.path": [
-    "${workspaceFolder}/.venv/bin/isort"
-  ],
-  "isort.args": [
-    "--settings-path=${workspaceFolder}/.isort.cfg"
-  ],
-  "python.analysis.autoImportCompletions": true,
-  "python.analysis.typeCheckingMode": "basic",
   "ruff.lint.args": [
```

Comment:
question: why is shellcheck disapearing here?
and isort? is that also intended?

Reply:
@sanderegg No shellcheck moved since I sorted them alphabetically and isort is replaced by ruff